### ISSUE TITLE
Midi.js cannot be minified

### ISSUE THREAD

**Issue by borisrorsvort** [paulrosen/abcjs#140]
Using create-react-app and abcjs 3.3.4

i’m getting error on production build 

```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/abcjs/midi.js:4
```
Read more here: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify

The es6 import works fine on dev mode when not trying to minify

**Comment by paulrosen** [comment]
Ok, I'll look into this tomorrow. I assume that you used `import abcjs from 'abcjs/midi';`. Does the same thing happen with `import abcjs from 'abcjs';`?

**Comment by borisrorsvort** [comment]
I dont think you should revert your code like that. What the docs said is that the module that is mapped for the default export should be an es5 build of the original source code. That give you the freedom to go nuts on es6 while keeping browser compat and be minifying safe

**Comment by paulrosen** [comment]
Fixed in 4.0.0
